Source/ALDataRequestView.swift

```diff
@@ -141,6 +139,7 @@ public class ALDataRequestView: UIView {
         loadingView?.removeFromSuperview()
         emptyView?.removeFromSuperview()
         reloadView?.removeFromSuperview()
+        hidden = true
```

Comment:
Als er niets of iets geladen wordt, wordt `**changeRequestState**` aangeroepen. Hierin wordt voor elke state de `**resetToPossibleState**` aangeroepen. 

Omdat `**hidden = true**` nu bij `**resetToPossibleState**` is toegevoegd, kan je ook de **dataRequestView** gebruiken als je nog geen reload view hebt toegevoegd. Anders krijg je de warning en kan de gebruiker niets meer doen want de **dataRequestView** is dan nog steeds zichtbaar **(hidden = false)**. 

In de `**showLoading**`, `**showReload**` en `**showEmpty**` wordt `**hidden**` op `**false**` gezet, dus dit blijft gewoon werken.